File: core/src/main/java/net/minestom/server/utils/math/ByteRange.java
package net.minestom.server.utils.math;

public class ByteRange extends Range<Byte> {

    public ByteRange(Byte minimum, Byte maximum) {
        super(minimum, maximum);
    }

    public ByteRange(Byte value) {
        super(value);
    }

    /**
     * {@inheritDoc}
     */
    @Override
    public boolean isInRange(Byte value) {
        return value >= this.getMinimum() && value <= this.getMaximum();
    }
}

File: src/main/java/net/minestom/server/utils/math/ByteRange.kt
package net.minestom.server.utils.math

class ByteRange : Range<Byte> {
    constructor(minimum: Byte, maximum: Byte) : super(minimum, maximum) {}
    constructor(value: Byte) : super(value) {}

    /**
     * {@inheritDoc}
     */
    override fun isInRange(value: Byte): Boolean {
        return value >= minimum && value <= maximum
    }
}